src/db/models/Article.js

```diff
@@ -30,9 +30,9 @@ export default (sequelize, DataTypes) => {
         type: DataTypes.STRING,
         allowNull: false,
       },
-      title: {
-        type: DataTypes.STRING,
-        allowNull: false,
+      tags: {
```

Comment:
Since we are using a join table to track the tags every article has which you have called ```ArticleTags```. IMO, it renders this column un-necessary and redundant.